### PR TITLE
appengine/bigquery: use google.DefaultClient, cleanup

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

**Comment by broady** [comment]
@campoy PTAL

File: appengine/bigquery/README.md [hunk 1]
```diff
@@ -2,16 +2,26 @@
...
-- get all the dependencies for this code snippet
+1. Get the dependencies:
+
+        goapp get .
+
+
+1. Run the app locally:
 
-	goapp get .
+        dev_appserver.py -A [your-project-id] .
```
Comment (campoy) [inline appengine/bigquery/README.md]: why not goapp serve?

**Comment by campoy** [comment]
Tiny detail, but other than that it looks good

File: appengine/bigquery/README.md (hunk 1 above)
Comment (broady) [inline appengine/bigquery/README.md]: goapp serve doesn't support -application :(

Comment (campoy) [inline appengine/bigquery/README.md]: Is there a bug open for that? It should

**Comment by campoy** [comment]
LGTM, but can't merge because your branch is not up to date

**Pull request merged**